packages/ui-components/src/components/text-field/text-field.types.ts

```diff
@@ -66,4 +66,8 @@ export interface ITextField {
 	forcedFocus?: boolean;
 	/** (optional) Text field tooltip configuration */
 	tooltipConfig?: Partial<ITooltip>;
+	/** (optional) Use a input mask when the text field type is number (default true) */
+	useInputMask?: boolean;
```

Comment:
would it be possible to infer the value of this boolean by checking if the `inputMaskRegex` is set?

Reply:
Not possible because the inputMaskRegex is optional and doesn't need to be used to apply the inputmask. It was added just to cover some more use cases if needed but for the purpose of this PR, which was to cover the possibility of having empty strings, its not used.